File: 프로그래머스/unrated/134240. 푸드 파이트 대회/푸드 파이트 대회.kt
class Solution {
    fun solution(food: IntArray): String {
        val answer = StringBuilder()

        for(i in 1 until food.size) {
            for(j in 0 until food[i] / 2) {
                answer.append(i)
            }
        }

        answer.append(0)

        for(i in food.size-1 downTo 1) {
            for(j in food[i]/2-1 downTo 0) {
                answer.append(i)
            }
        }

        return answer.toString()
    }
}

File: 프로그래머스/1/134240. 푸드 파이트 대회/푸드 파이트 대회.swift
func solution(_ food:[Int]) -> String {
    var answer = ""
    
    for i in 1...food.count-1 {
        if food[i] != 1 {
            for j in 1...food[i]/2 {
                answer += "\(i)"
            }
        }
    }
    
    answer += "0"
    
    for i in stride(from: food.count-1, to: 0, by: -1) {
        if food[i] != 1 {
            for j in 1...food[i]/2 {
                answer += "\(i)"
            }
        }
    }
    
    return answer
}